.github/workflows/zeebe-version-compatibility.yml

```diff
@@ -43,6 +43,8 @@ jobs:
           secret_vault_address: ${{ secrets.VAULT_ADDR }}
           secret_vault_roleId: ${{ secrets.VAULT_ROLE_ID }}
       - uses: ./.github/actions/build-zeebe
+        with:
+          maven-extra-args: -T1C -PskipFrontendBuild
```

Comment:
:upside_down_face: `-T1C` is the default already

Reply:
I just copy-pasted it from another workflow 😄 But you're right, I'll remove it.